.github/workflows/continous-integration.yml

```diff
@@ -139,12 +139,18 @@ jobs:
       with:
```

Comment:
general question: Do we actually need to install `wget`? Apparently it's already included https://github.com/agda/agda/issues/4691

Reply:
Might be true. It's there because I haven't found it on this list (https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md) and because it was in our `Makefile` before.
I will try removing it.

Reply:
Apparently, no, it doesn't work: https://github.com/RasaHQ/rasa/pull/6403/checks?check_run_id=1123537533

Reply:
thanks for trying it out 👍